src/components/PDFExport/index.tsx

```diff
@@ -101,11 +100,7 @@ const PDFExport = ({
     <div className="easi-pdf-export" ref={divEl}>
       {linkPosition === 'bottom' && children}
 
-      <div
-        className={classNames('easi-pdf-export__controls', {
-          'margin-top-6': linkPosition === 'bottom'
-        })}
-      >
+      <div className="easi-pdf-export__controls">
```

Comment:
This change undoes work from https://github.com/CMSgov/easi-app/pull/1667; please manually reverse it.